model_analyzer/triton/model/model_config.py

```diff
@@ -149,8 +150,7 @@ def cpu_only(self):
 
         return self._cpu_only
 
-    def write_config_to_file(self, model_path, src_model_path,
-                             last_model_path):
+    def write_config_to_file(self, model_path, src_model_path, last_model_path):
```

Comment:
Why do we still have formatting changes after the format PR was merged? Is this branch not rebased on `main`?

Reply:
Yes it is not rebased, it will be